src/resample/bootstrap.py

```diff
@@ -111,12 +111,12 @@ def jackknife_bias_corrected(a: np.ndarray, f: Callable) -> np.ndarray:
         Sample. Must be one-dimensional.
 
     f : callable
-        Estimator. Can be any mapping R^N -> R^M, where N is the number of samples.
+        Estimator. Can be any mapping ℝⁿ → ℝᵐ, where n is the number of samples.
```

Comment:
+1 on the rendering of R^n -> R^m

Reply:
Actually, the "m" is a little hard to read (it looks a lot like the "n", so it may confuse people into thinking this is actually R^n -> R^n). What do you think? I think we could at least add some text like "and m is a positive integer" to bring attention to the fact that there are two different values here.

Also whichever we choose we probably want to do the same in the docstring for jackknife itself (currently it's rendered differently).

Reply:
Good points, maybe I should use `n` and `k`. Using unicode should be fine, it is supported everywhere nowadays and since we are not targeting Python2, I think it is safe to use here. I would try it out and roll back if people complain.

Reply:
Ah, I think I used m because superscript letters do not exist for all characters in unicode.